app/lib/features/pins/pin_utils/pin_utils.dart

```diff
@@ -0,0 +1,174 @@
+import 'dart:io';
+import 'dart:typed_data';
+import 'package:acter/common/utils/utils.dart';
+import 'package:acter/common/widgets/attachment_options.dart';
+import 'package:acter/features/pins/providers/pins_provider.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:file_picker/file_picker.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:mime/mime.dart';
+
+typedef PinAttachment = ({AttachmentType type, File file});
+
+class PinUtils {
```

Comment:
this is somewhat odd thing to have ... and additionally in a `pin/pin_utils/pin_utils.dart`-file ... shouldn't this just be on some kind of widget directly?

Reply:
Let's keep it this way for now and I will refactor it in subsequent PR's ;)